apireader.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+
+import sys
+import os.path
+import requests
+import json
+from argparse import ArgumentParser
+
+DUMP_NAME = "api_data.dump"
+
+
+def get_api_dict(src_dir):
+    with open(src_dir + "directory.json", 'r') as directory_file:
+        api_dict = json.load(directory_file)
+    return api_dict
+
+
+def get_api_data_from_git(api_dict):
+    data = dict()
+    for community in api_dict:
+        api_url = api_dict[community]
+        print("Loading %s..." % api_url, file=sys.stderr)
```

Comment:
Don't use string interpolation, instead use string formatting. Printing the community would also be useful...

```
        print("Loading {:<16} {}...".format(community, api_url), file=sys.stderr)
```